### PR TITLE
A Gentle Introduction to the Python Binance API

### LINKED ISSUE #4665: [Machine Learning] A Gentle Introduction to the Python-Binance API
#### Proposed title of article
Your title should be descriptive of the article/tutorial. Be Specific. 

A Gentle Introduction to the Python-Binance API

#### Proposed article introduction
*We're looking for the first 2-3 paragraphs of your article that appropriately summarize what your article will be about.*
We will use this writing snippet to help us assess the overall quality before approval so ensure it has been proofread, spellchecked and grammarchecked.

Binance is a cryptocurrency exchange which is currently the largest exchange in the world in terms of daily trading volume of cryptocurrencies. The Binance API is a method that allows you to connect to the Binance servers via Python or several other programming languages.
This tutorial will give you a gentle introduction to the Python-Binance API. Later, we'll show you how to apply ML to trading and finance using this library.

#### Key takeaways 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
Use this area to get your ideas down on the bulk of your article or tutorial.

By the end of this tutorial, a reader will:
- Be introduced to the Python-Binance API.
- Learn about python-binance for crypto trading and finance.
- Learn how to install and import python-binance
- Learn how to apply ML to trading and finance using Python-Binance.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.
- [Python-Binance](https://python-binance.readthedocs.io/en/latest/)

### PR DESCRIPTION
**Pull request by wilkistermumbi**
This closes #4665

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

**Comment by hectorkambow** [comment]
@collins-a 

You may be working on it - just wanted to get a status on this PR. 

Would like to get it cleared out of the queue asap 🚀

File: content/articles/a-gentle-introduction-to-the-python-binance-api/index.md [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+
+Python-binance is an application programming interface that allows you to connect to the Binance servers via the python programming language. It is important to note that the python-binance library is not affiliated with Binance. With the API, you can make orders, trade, withdraw and get real-time data from the Binance exchange. To learn more about the library, feel free to read their [documentation](https://python-binance.readthedocs.io/en/latest/index.html).
+
+### Setting up the binance API
+We first need to set up an API key by registering an account on Binance. This step involves heading to the Binance [website](https://www.binance.com/en) and signing up if you don't have an account with them yet. Registration is straightforward, as you would register for any web application. After successfully signing up and completing the verification process, hover over the profile button and select the `API Management` option as shown below:
```
Comment (collins-a) [inline content/articles/a-gentle-introduction-to-the-python-binance-api/index.md:41]: Consider removing the second line in this paragraph.

File: content/articles/a-gentle-introduction-to-the-python-binance-api/index.md [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+```python
+apikey = 'ENTERYOURAPIKEY'
+apisecret = 'ENTERYOURSECRETKEY'
+```
+We've stored our API and secret keys in the `apikey` and `apisecret` variables.
+
+### Installing and importing dependencies
+In this next step, we'll go ahead and install and import the necessary dependencies.
+
+```python
```
Comment (collins-a) [inline content/articles/a-gentle-introduction-to-the-python-binance-api/index.md:60]: _Bash_ in place of _python_

File: content/articles/a-gentle-introduction-to-the-python-binance-api/index.md [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+Output:
+
+![Visualization](/engineering-education/a-gentle-introduction-to-the-python-binance-api/plot.png)
+
+As we've seen, the matplotlib finance library allows us to visualize data easily.
+
+Please find the code for this tutorial [here](https://colab.research.google.com/drive/1MP-Kk4gG-fnbCK40MxENCdIKr0KRho9f?usp=sharing).
+
+### Summary
+In a nutshell, this tutorial has shown you what's possible with the python-binance API. We've shown you how you can perform an exploratory data analysis on Binance data. To avoid making the tutorial so long, we've only limited it to the EDA bit. In our follow-up tutorial, we'll build upon this tutorial and show you how to apply machine learning and data science to trading and finance.
```
Comment (collins-a) [inline content/articles/a-gentle-introduction-to-the-python-binance-api/index.md:332]: The third sentence in this paragraph seems off. Consider restructuring it or removing it entirely.

**Review by collins-a — COMMENTED** [review]
Well done @wilkistermumbi. handle the minor changes highlighted and consider adding definitions or a link to the trading jargon (such as _spot_ and _futures_ among others) covered in the article, as well as a prerequisites section. Ping me when done.

**Comment by wilkistermumbi** [comment]
@collins-a  I'm done making the changes.

**Review by collins-a — APPROVED** [review]

**Comment by collins-a** [comment]
> @collins-a
> 
> You may be working on it - just wanted to get a status on this PR.
> 
> Would like to get it cleared out of the queue asap 🚀

@hectorkambow I'm done reviewing and it's ready for the final review.

**Comment by lalith1403** [comment]
Jumping on this one.

**Review by lalith1403 — APPROVED** [review]
Easy to follow and well written article @ . Well done on the review process @collins-a. Great work.

**Pull request merged**